htdocs/modules/system/themes/modern/modern.php

```
@@ -0,0 +1,433 @@
+                $info = $mod->getInfo();
+
+                $item = [];
+                $item['name'] = $mod->getVar('name');
+                $item['dirname'] = $mod->getVar('dirname');
+                $item['icon'] = isset($info['image']) ? XOOPS_URL . '/modules/' . $mod->getVar('dirname') . '/' . $info['image'] : '';
+
+                if (!empty($info['adminindex'])) {
+                    $item['link'] = XOOPS_URL . '/modules/' . $mod->getVar('dirname', 'n') . '/' . $info['adminindex'];
+                } else {
+                    $item['link'] = XOOPS_URL . '/modules/system/admin.php?fct=preferences&amp;op=showmod&amp;mod=' . $mod->getVar('mid');
+                }
```

PHPMD (2.15.0)

[error] The method buildMenu uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)